packages/core/src/ObjectContainer.js

```diff
@@ -815,4 +878,13 @@ class Entry {
   get writeable() {
     return this._options.writeable;
   }
+
+  static createFromEntry(entry) {
```

Comment:
I would choose 'Entry.from' instead of 'Entry.createFromEntry' similar to Array.from